docs/src/user/manual/profile-page.md

```diff
@@ -0,0 +1,28 @@
+# SAWPS profile page
+
+The Profile tab is the first tab on the profile configuration page. The page consists of the following components with the following functionality. 
+
+![Profile Page](img/profile.png)
+
+1. The profile tab - a tab that displays the user profile information
+2. Profile picture - an area that displays the uploaded profile image
+3. Upload profile picture link - Opens the upload profile picture popup (see profile picture pop-up)
+4. Update password button - Opend the update password form for the user to make a new password for the platform (see update password form)
```

Comment:
```suggestion
4. Update password button - Opens the update password form for the user to make a new password for the platform (see update password form)
```

Reply:
Thanks for the eagle eye. I will update that and PR